multiple_load_object_wrap/node-addon-api/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "object_wrap",
+  "version": "0.0.0",
+  "description": "Multiple load example of ObjectWrap",
+  "main": "addon.js",
+  "private": true,
+  "gypfile": true,
+  "engines": {
+    "node": ">= 14.0.0"
+  },
```

Comment:
This needs to work with all supported versions of Node.js.

Reply:
@anfilat using the value for "engiens" as suggested at https://github.com/nodejs/node-addon-examples/pull/139#issuecomment-637896592 should make the CI green, and it should actually test your code on Travis at least, even if it doesn't test it on GitHub actions.

Reply:
Oh, wait, #139 needs to land to update the .travis.yml removing the non-supported Node.js 8 first 🙂

Reply:
@gabrielschulhof I think I should close this PR. It's duplicate with https://github.com/nodejs/node-addon-examples/pull/139 now